src/wallet/scriptpubkeyman.h

```diff
@@ -631,6 +634,18 @@ class DescriptorScriptPubKeyMan : public ScriptPubKeyMan
 
     void UpgradeDescriptorCache();
 };
+
+/** struct containing information needed for migrating legacy wallets to descriptor wallets */
+struct MigrationData
```

Comment:
NACK: this struct should be removed from `src/wallet/walletutil.h`.

maybe missing backport